app/presenters/nav_tab/link_collection.rb

```diff
@@ -38,10 +38,21 @@ def default
 
   private
 
+  def payment_sources
```

Comment:
I don't like that naming convention only because none of the other navlinks methods use it. If we change this name to `my_payment_sources`, I'd want to change all the method names referenced in line 32 above.